src/components/modal/v2/styles/components/_calculator.scss

```
@@ -110,6 +110,18 @@
             }
```

Making sure the input field width matches the figma.
```suggestion
            }
            
             .v5Design & {
                width: 98%;
            }
```